packages/flutter/lib/src/cupertino/route.dart

```diff
@@ -58,6 +58,11 @@ const Color kCupertinoModalBarrierColor = CupertinoDynamicColor.withBrightness(
 // The duration of the transition used when a modal popup is shown.
 const Duration _kModalPopupTransitionDuration = Duration(milliseconds: 335);
 
+/// The duration of the page transition used for [CupertinoPageRoute].
+///
+/// A relatively rigorous eyeball estimation.
+const Duration kCupertinoPageTransitionDuration = Duration(milliseconds: 500);
```

Comment:
This should probably be a static property in CupertinoRouteTransitionMixin to be more inline with other transition related property that shared between cupertino and material